src/collections/_documentation/data-management/rollups.md

```diff
@@ -74,3 +70,200 @@ A generic error, such as a database connection error, has many different stack t
+single enhancement rule. It's one or multiple match expressions followed by one
+or multiple actions to be executed when all expressions match. All rules are
+executed from top to bottom on all frames in the stack trace.
+
+The syntax for grouping enhancements is roughly like this:
+
+```
+matcher-name:expression other-matcher:expression ... action1 action2 ...
+```
+
+#### Rules
+
+the following **matchers** exist. Multiple matchers can be defined in a line:
+
+- `family`: matches on the general platform family. Right now there are `javascript`, `native`
+  and `other`. To make a rule apply to multiple platforms you can comma separate them.
+  Eg: `family:native,javascript` applies to both javascript and native.
+- `path`: this matches case insensitive with unix glob behavior on a path. The path
+  separators are normalized to `/`. As a special rule if the filename is relative it
+  still matches on `**/`. Examples:
+  - `path:**/project/**.c`: matches on all files under `project` with a `.c` extension
+  - `path:**/vendor/foo/*.c`: matches on vendor/foo without sub folders
+  - `path:**/*.c`: matches on `foo.c` as well as `foo/bar.c`.
+- `module`: is similar to `path` but matches on the module. This is not used for native
+  but for instance for javascript, python and similar platforms. Matches are
+  case sensitive and normal globbing is available.
+- `function`: matches on a function, case sensitive with normal globbing.
+- `function:myproject_*` matches all functions starting with `myproject_`
+- `function:malloc` matches on the malloc function
+- `package`: matches on a package. The package is the container a function or module is
+  contained in. This is a `.jar`, a `.dylib` or similar. The same matching rules as for
+  `path` apply (eg: this is an absolute path typically).
+- `app`: matches on the current state of the in-app flag. `yes` means the frame is in app, `no` means it's not.
+- An expression can be quoted if necessary (eg: spaces are included)
+
+There are two types of **actions**: flag setting and setting variables.
+
+- **flag**: flags are what is to be done if all matchers match. a flag needs to be prefixed with `+` to set
+  it or `-` to unset it. If this expression is prefixed with a `^` it applies to frames above (eg:
+  towards the crash) the frame, if prefixed with `v` it applies to frames below
+  (eg: away from the crash). For instance `-group ^-group` removes the matching frame
+  and all frames above from the grouping.
+  - `app`: marks or unmarks a frame in-app
+  - `group`: adds or removes a frame from grouping
+- **variables**: additionally variables can be set (`variable=value`). Currently there is just one:
+  - `max-frames`: sets the total number of frames to be considered for grouping.
+    The default is `0` which means "all frames". If set to `3` only the top
+    3 frames are considered.
+
+If a line is prefixed with a hash (`#`) it's a comment and ignored
+
+#### Examples
```

Comment:
Can we provide the example also with the stack trace? So that we can visualise how the rules are applied.

Reply:
Can you elaborate what you mean here?